src/main/scala/com/ing/wbaa/rokku/proxy/provider/aws/S3Client.scala

```diff
@@ -35,12 +38,14 @@ trait S3Client {
    * @param bucketName The name of the bucket to set the policy
    * @return A future which completes when the policy is set
    */
-  protected[this] def setDefaultBucketAclAndPolicy(bucketName: String): Future[Unit] = Future {
+  protected[this] def setDefaultBucketAclAndPolicy(bucketName: String)(implicit id: RequestId): Future[Unit] = Future {
+    logger.info("setting bucket acls and policies for bucket {}", bucketName)
     val acl = s3Client.getBucketAcl(bucketName)
     acl.grantPermission(GroupGrantee.AuthenticatedUsers, Permission.Read)
     acl.grantPermission(GroupGrantee.AuthenticatedUsers, Permission.Write)
     s3Client.setBucketAcl(bucketName, acl)
     s3Client.setBucketPolicy(bucketName, """{"Statement": [{"Action": ["s3:GetObject"],"Effect": "Allow","Principal": "*","Resource": ["arn:aws:s3:::*"]}],"Version": "2012-10-17"}""")
+    logger.info("acls and policies for bucket {} done", bucketName)
```

Comment:
Perhaps, we could wrap setBucketPolicy in Try and log if SdkClientException is not thrown?

Reply:
done